configure.ac

```diff
@@ -959,7 +959,7 @@ fi
 # Linters
 AC_CHECK_PROG([flake8],[flake8],[yes],[no])
 AM_CONDITIONAL([FOUND_FLAKE8], [test "x$flake8" = xyes])
-AC_CHECK_PROG([cpplint],[cpplint.py],[yes],[no])
+AC_CHECK_PROG([cpplint],[cpplint],[yes],[no])
```

Comment:
Ah this might make my backwards compatible comment a bit less relevant.

Reply:
I think we could also completely take this and flake8 out of dependency checking, if you're OK with that. I don't think that someone building OLA should necessarily be required to have cpplint and flake8 (or maybe there is already a `./configure` flag for that).

Reply:
I don't believe it should block the build if they're not present.

Indeed my config.log has:
```
configure:27245: checking for flake8
configure:27273: result: yes
configure:27291: checking for cpplint.py
configure:27319: result: no
```

It just means we can present a more user-friendly message if they try to run the tools.